flatdata-rs/tests/coappearances/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "tests_coappearences"
```

Comment:
Should not this package called something with flatdata prefix?

Reply:
Done